x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -683,6 +683,13 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                 // Fleet Server indices. Kibana create this indice before Fleet Server use them.
                 // Fleet Server indices. Kibana read and write to this indice to manage Elastic Agents
                 RoleDescriptor.IndicesPrivileges.builder().indices(".fleet*").allowRestrictedIndices(true).privileges("all").build(),
+                // Endpoint specific action responses. Kibana reads from these to display responses to the user.
+                RoleDescriptor.IndicesPrivileges.builder().indices(".logs-endpoint.action.responses-*").privileges("read").build(),
+                // Endpoint specific actions. Kibana reads and writes to this index to track new actions and display them.
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(".logs-endpoint.actions-*")
+                    .privileges("create_index", "auto_configure", "read", "write")
+                    .build(),
```

Comment:
We can remove `auto_configure` since it is covered by `create_index` and `write`.

Similar to the above comment, I'd prefer we relocate this statement along with the above one to under line 709-720.

Reply:
removed `auto_configure` and moved the statement: https://github.com/elastic/elasticsearch/pull/81953/commits/f703c23577178435be56e269ec4afcd6d162ecd6

Reply:
updating this, based on feedback further down, I'm using `auto_configure` as opposed to `create_index`